README.md

```diff
@@ -3,63 +3,70 @@
+
+## Clean architecture with 3 main modules
 - Data (for database, API and preferences code)
 - Domain (for business logic and models)
-- Presentation (for UI logic, with MVVM)
+- AndroidApp (for UI logic, with MVVM)
 
- <img src="images/AndroidTemplate-CleanArchitecture.png" alt="ArchiTecture logo"/>
+ <img src="images/AndroidTemplate-CleanArchitecture.jpg" alt="ArchiTecture logo"/>
 
 ## Tests
+- [Mockk](https://mockk.io/) library 
 - Unit tests
 - Application tests
-- Activity tests (with [Espresso](https://google.github.io/android-testing-support-library/docs/espresso/))
-- Application has a testing flag
+  - example on how to work with tests
+- Activity tests (with [Compose Testing](https://developer.android.com/jetpack/compose/testing))
+  - example on how to work with coroutine scopes in tests
     
 ## Other useful features
 - Dependency injection (with [Hilt](http://google.github.io/hilt/))
+- Network calls (with [Ktor](https://ktor.io/docs/http-client-engines.html#minimal-version))
 - Reactive programming with [Kotlin Flows](https://kotlinlang.org/docs/reference/coroutines/flow.html)
-- Google [Material Design](https://material.io/blog/android-material-theme-color) library
-- Margins and Insets with [Insetter](https://chrisbanes.github.io/insetter/)
-- Logging (with [Timber](https://github.com/JakeWharton/timber))
 - Android architecture components to share ViewModels during configuration changes
+- Google [Material Design](https://material.io/blog/android-material-theme-color) library
+- UI with [Jetpack Compose](https://developer.android.com/jetpack/compose)
+- Compose Navigation (with [Hilt Support](https://developer.android.com/jetpack/compose/libraries#hilt-navigation) and Assisted Inject Example)
 - Edge To Edge Configuration
-- Resource defaults
-    - themes.xml - app themes
-    - themes.xml (v29) - app themes (for better edgeToEdge)
-    - colors.xml - colors for the entire project
-    - styles.xml - widget styles 
-    - styles-text.xml - text appearances
+- [Splash Screen](https://developer.android.com/develop/ui/views/launch/splash-screen) Support - 
+- [Version Catalog](https://docs.gradle.org/current/userguide/platforms.html)
 
 # Getting started
 
 ### Script 
-1. Download this repository extract and open the folder
-2. Run `newproject.sh` script to create a copy with the name you want 
-    * Example: `./newproject.sh -p com.example.project -t Project` 
-3. On `app/build.gradle`, update the dependencies Android Studio suggests
-4. On `theme.xml` & `colors.xml` set your application primary and secondary colors 
-
+1. Run the Project NewProjectCI
+2. On `Theme.kt` & `Color.kt` set your application style
+3. Replace the App Icons
+4. Delete unwanted example files 
+5. Run `./gradlew dependencyUpdates` and check for dependencies
+6. Ready to Use
 
 And you're ready to start working on your new app.
 
 ### Manual
```

Comment:
We should merge the Script and the Manual into a single set of instructions. Since there are a lot of instructions, it might simpler to read if we have some high-level instructions and then some nested instructions inside each one.

Reply:
...? merge any suggestion how? not sure if I understand